plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/ReadFunction.java

```diff
@@ -15,11 +15,17 @@
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.Optional;
 
 public interface ReadFunction
 {
     Class<?> getJavaType();
 
+    default Optional<String> getBindExpression(String columnName)
+    {
+        return Optional.empty();
```

Comment:
Let's have default implementation which returns `columnName`.
Of course, this requires that we pass in a quoting function, but implementations will need anyway (any generic implementation; concrete impls can assume eg `"`-quoting)